types/liftoff/index.d.ts

```diff
@@ -17,65 +18,58 @@ declare class Liftoff extends EventEmitter {
      */
     constructor(options?: Liftoff.Options);
 
-    /**
-     * Launches your application with provided options, builds an environment,
-     * and invokes your callback, passing the calculated environment as the first argument.
-     */
-    launch(
-        opts: Liftoff.LaunchOptions,
-        callback: (this: this, env: Liftoff.LiftoffEnv) => void
-    ): void;
+    prepare(options: Liftoff.PrepareOptions, callback: (env: Liftoff.LiftoffEnv) => void): void;
+    execute(env: Liftoff.LiftoffEnv, callback: (env: Liftoff.LiftoffEnv) => void): void;
 
     addListener(
-        event: 'require',
+        event: 'preload:success',
         listener: (name: string, module: ExtensionDescriptor) => void
     ): this;
-    addListener(event: 'requireFail', listener: (name: string, err: any) => void): this;
+    addListener(event: 'preload:failure' | 'preload:before', listener: (name: string, err: any) => void): this;
```

Comment:
- The `preload:before` event doesn't pass an `err` parameter.
- The `err` parameter's type can be changed to `Error`